tests/test_cli.sh

```diff
@@ -8,6 +8,12 @@ function setup_tests {
     # explicitly enable sshd for live-iso b/c it is disabled by default
     # due to security concerns (no root password required)
     sed -i.orig 's/^services.*/services --disabled="network" --enabled="NetworkManager,sshd"/' $1/composer/live-iso.ks
+    # explicitly enable logging in with empty passwords via ssh
+    awk -i inplace "
+        /%post/ && FLAG != 2 {FLAG=1}
+        /%end/ && FLAG == 1 {print \"sed -i 's/.*PermitEmptyPasswords.*/PermitEmptyPasswords yes/' /etc/ssh/sshd_config\"; FLAG=2}
+        {print}" \
+        $1/composer/live-iso.ks
```

Comment:
Why do we need to explicitly enable this? IIRC it is possible to login into the live-iso VM without a password and that is why ssh is disabled by default. And also the existing live-iso tests seem to be working without this change so what causes the need for this ?

Reply:
We need to use the explicit `sshd` setting for `PermitEmptyPasswords` because it is disabled by default for security reasons and this option is not set (is commented out) in `sshd_config`. I also updated the comment before the respective `awk` code to be a bit more descriptive.
The default setting is stated in `man sshd_config` and is consistent with my observations when I tried to login to a machine with empty root password and default `sshd` settings. So I wonder what could be different in my case if your experience is different?

Reply:
This snippet needs to be removed, we are not enabling ssh login with empty passwords during testing.